Documentation/source/Integration.md

```diff
@@ -92,3 +92,42 @@ The components that can be found in the SDK are:
 * **Analysis**: Provides a UI for the analysis process of the document by showing the user capture tips when an image is analyzed or the document information when it is a PDF. In both cases an image preview of the document analyzed will be shown (`AnalysisViewController`).
 * **Help**: Helpful tutorials indicating how to use the open with feature, which are the supported file types and how to capture better photos for a good analysis (`HelpMenuViewController`).
 * **No results**: Shows some suggestions to capture better photos when there are no results after an analysis (`ImageAnalysisNoResultsViewController`).
+
+## Sending Feedback
+
+Your app should send feedback for the extractions the Gini Pay API delivered. Feedback should be sent only for the extractions the user has seen and accepted (or corrected).
+
+For additional information about feedback see the [Gini Pay API documentation](https://pay-api.gini.net/documentation/#send-feedback-and-get-even-better-extractions-next-time).
+
+### Default Implementation
+
+The example below shows how to correct extractions and send feedback using the default networking implementation:
+
+You should send feedback only for extractions the user has seen and accepted.
+
+
+```swift
+var sendFeedbackBlock: (([String: Extraction]) -> Void)?
+var extractions: [String: Extraction] = []
+
+func giniCaptureAnalysisDidFinishWith(result: AnalysisResult,
+                           sendFeedbackBlock: @escaping ([String: Extraction]) -> Void){
+        
+    self.extractions = result.extractions
+    self.sendFeedbackBlock = sendFeedbackBlock
+    showResultsScreen(results: result.extractions.map { $0.value })
+}
+.
+.
+.
+// In this example only the amountToPay was wrong and we can reuse the other extractions.
+let updatedExtractions = self.extractions
+updatedExtractions.map{$0.value}.first(where: {$0.name == "amountToPay"})?.value = "31,25:EUR"
+sendFeedbackBlock(updatedExtractions)
+
+```
+### Custom Implementation
+
+If you use your own networking implementation and directly communicate with the Gini Pay API then see [this section](https://pay-api.gini.net/documentation/#submitting-feedback-on-extractions) in its documentation on how to send feedback.
+
+In case you use the [Gini Pay API Library](https://developer.gini.net/gini-pay-api-lib-ios/docs/getting-started.html) then see this section in its documentation for details.
```

Comment:
You will add the link here to `this section` after you updated the Gini Pay API Library right? I would wait then until that's done before approving.

Reply:
The section name is valid -> still "getting-started"

Reply:
The problem is, that `see this section` doesn't point to anything. I would propose this change:
```
In case you use the [Gini Pay API Library](https://developer.gini.net/gini-pay-api-lib-ios/docs/index.html) then see [this section](https://developer.gini.net/gini-pay-api-lib-ios/docs/getting-started.html) in its documentation for details.
```

Reply:
Haha, you committed the exact change I was proposing right when I was writing it 😄